GoogleDataTransport/GDTCORLibrary/Internal/GDTCORStorageProtocol.h

```diff
@@ -128,4 +162,10 @@ typedef void (^GDTCORStorageBatchBlock)(NSNumber *_Nullable newBatchID,
 FOUNDATION_EXPORT
 id<GDTCORStorageProtocol> _Nullable GDTCORStorageInstanceForTarget(GDTCORTarget target);
 
+// TODO: Ideally we should remove completion-based API and use promise-based one. Need to double
+// check if it's ok.
```

Comment:
Stale comment?

Reply:
Well, not really... I still would prefer to avoid duplicating the API but on the other hand I also would prefer to limit scope of the PR to the minimal required set of changes. Because of that I'm leaving the TODO to address it later on.